backend/.env.example

```
@@ -1,2 +1,3 @@
-IP_ADD=YOUR_IP:PORT
-GPU=NONE #NVIDIA OR NONE
\ No newline at end of file
+IP_ADD=YOUR_IP
+KAFKA_BROKER=IP_WHERE_KAFKA_HOSTED:PORT
+GPU=NONE #NVIDIA OR INTEGRATED
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)